### PR TITLE
List all books on browsing to "/"

### PR DESCRIPTION
**Pull request by qptr**

### CODE REVIEW

File: test/clj/athenaeum/domain/book_test.clj [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    (let [_book1 (jdbc/with-transaction [tx @db/datasource]
+                   (book/create tx "book1" "author1"))
+          _book2 (jdbc/with-transaction [tx @db/datasource]
+                   (book/create tx "book2" "author2"))
+          _book3 (jdbc/with-transaction [tx @db/datasource]
+                   (book/create tx "book3" "author3"))
+          res (jdbc/with-transaction [tx @db/datasource]
+                (book/fetch-all tx))]
+      (is (= 3 (count res)))
+      (is (= "book1" (:books/title (first res))))
```
Comment (jysandy) [inline test/clj/athenaeum/domain/book_test.clj]: We should assert on the entire return value here.

**Review by jysandy — CHANGES_REQUESTED** [review]

**Comment by qptr** [comment]
Looks like this with a dummy link
Screen Shot 2021-01-04 at 16 11 21

and like this without
Screen Shot 2021-01-04 at 16 12 47

File: src/cljs/athenaeum_web/routes.cljs [hunk 2]
```diff
@@ -5,15 +5,22 @@
             [athenaeum-web.events :as e]))
 
 (def routes
-  ["/" [["" :home-page]
+  ["/" [[""   :home-page]
         [true :not-found]]])
 
+(def page-list
+  [{:handler :home-page :page-name "Home"}])
```
Comment (jysandy) [inline src/cljs/athenaeum_web/routes.cljs]: Let's move this to be alongside the navbar.

File: test/clj/athenaeum/fixtures.clj [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+(defn set-datasource
+  [f]
+  (let [previous-datasource @db/datasource]
+    (db/set-datasource)
+    (f)
+    (db/reset-datasource previous-datasource)))
+
+(defn clear-tables
+  [f]
+  (jdbc/execute! @db/datasource ["TRUNCATE TABLE books"])
```
Comment (jysandy) [inline test/clj/athenaeum/fixtures.clj]: We can use the `clear-tables` function from test-utils here.

**Review by jysandy — CHANGES_REQUESTED** [review]
LGTM aside from the suggested changes.

**Review by jysandy — APPROVED** [review]

**Pull request merged**